CMakeLists.txt

```diff
@@ -151,8 +151,6 @@ endif()
 set(GZ_RENDERING_ENGINE_RELATIVE_INSTALL_DIR
   ${GZ_LIB_INSTALL_DIR}/gz-${GZ_DESIGNATION}-${PROJECT_VERSION_MAJOR}/engine-plugins
 )
-set(GZ_RENDERING_ENGINE_INSTALL_DIR
-  ${CMAKE_INSTALL_PREFIX}/${GZ_RENDERING_ENGINE_RELATIVE_INSTALL_DIR})
```

Comment:
I think we may still need to keep this cmake variable here as I see that these are being used in [ogre/src/CMakeLists.txt](https://github.com/gazebosim/gz-rendering/blob/33626b5c48c20d147c167e64fa117043aceec5a5/ogre/src/CMakeLists.txt#L65-L66) and [ogre2/src/CMakeLists.txt](https://github.com/gazebosim/gz-rendering/blob/33626b5c48c20d147c167e64fa117043aceec5a5/ogre2/src/CMakeLists.txt#L90-L91)

Reply:
Can we replace these calls with `getResourcePath()`?

Reply:
`getResourcePath()` is a C++ function so we won't be able to replace those calls in CMakeLists.txt. I think we can just keep this cmake var here. Other changes in this PR are fine.

Reply:
Sorry I meant `GZ_RENDERING_RESOURCE_PATH`, that it's used in some `.cc` files (and in `optix/src/CMakeLists.txt`).

Reply:
to clarify, I think we are still keeping the environment variable and only deprecating the macro? Or are we deprecating / removing both?

Reply:
Sorry for the confusion. I think what we want is to remove just the `GZ_RENDERING_RESOURCE_PATH` macro and nothing else. We'll keep supporting the environment variables. [6bd5db5](https://github.com/gazebosim/gz-rendering/pull/1091/commits/6bd5db53959eb69536437e44a1a3fcb8049662d9)